compilation.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-npx buidler compile
 #npx buidler compile --config buidlerConfigV4.js <-- waiting for buidler fix
-node --max-old-space-size=4096 node_modules/.bin/buidler compile --config buidlerConfigV4.js
\ No newline at end of file
+node --max-old-space-size=4096 node_modules/.bin/buidler compile --config buidlerConfigV4.js
+npx buidler compile
```

Comment:
extra line please

so no we compile v4 first?
what is the motivation for this change?

Reply:
Used to have KyberNetwork contracts in both v4 and v5. v4 would have overwritten the compiled v5 one.

I think now that u removed the old contracts, it should work without needing this change. Just need to test.

Reply:
Yup, works. Reverting.